edge-modules/TwinTester/Program.cs

```diff
@@ -18,35 +18,61 @@ static async Task Main()
         {
             Logger.LogInformation($"Starting twin tester with the following settings:\r\n{Settings.Current}");
 
+            Console.WriteLine(StatusCode.DesiredPropertyReceived.ToString());
+
             try
             {
                 RegistryManager registryManager = RegistryManager.CreateFromConnectionString(Settings.Current.ServiceClientConnectionString);
```

Comment:
Should we have both the registry manager and module client initialization in one place? I'm assuming you didn't move registry manager init to GetTwinOperationAsync because needed in all cases?

Reply:
registryManager is needed all the time, but moduleClient is not.